randovania/games/prime1/json_data/header.json

```diff
@@ -1092,7 +1092,7 @@
                                         "type": "resource",
                                         "data": {
                                             "type": "items",
-                                            "name": "Scan",
```

Comment:
?

Reply:
Can't tell what the issue is here exactly.